crates/sui-indexer-alt-e2e-tests/tests/graphql/transaction_effects/execution_error.move

```diff
@@ -84,9 +91,11 @@ module test::execution_error_tests {
 
 //# run test::execution_error_tests::abort_with_clever_string --sender B
 
-//# run test::execution_error_tests::abort_with_clever_code --sender A
+//# run test::execution_error_tests::abort_with_clever_with_code --sender A
```

Comment:
Looks like you accidentally changed the function name called causing the execution to fail with "Function Not Found" instead of clever code.
```suggestion
//# run test::execution_error_tests::abort_with_clever_code --sender A
```